packages/documentation-site/patternfly-docs/content/design-guidelines/styles/typography/typography.md

```diff
@@ -48,101 +47,84 @@ The following sections outline common text styles that are used in PatternFly ap
   asGrid
   styleProps={styleProps.hero} />
-<div class="pf-v6-u-mt-lg">
-</div>
+<div class="pf-v6-u-mt-lg"></div>
 
 ### First level heading (2xl, H1)
 
 **Note**: Only 1 first level/H1 heading should ever exist on a page.
 
 <TitleLevel
-  className="pf-v6-c-title pf-m-lg ws-title-level-heading"
+  className="pf-v6-c-title pf-m-lg"
   asGrid
   styleProps={styleProps.first} />
-<div class="pf-v6-u-mt-lg">
-</div>
+<div class="pf-v6-u-mt-lg"></div>
 
 ### Second level heading (xl, H2)
 
 <TitleLevel
-  className="pf-v6-c-title pf-m-lg ws-title-level-heading"
+  className="pf-v6-c-title pf-m-lg"
   asGrid
   styleProps={styleProps.second} />
-<div class="pf-v6-u-mt-lg">
-</div>
+<div class="pf-v6-u-mt-lg"></div>
 
 ### Third level heading (lg, H3)
 
 <TitleLevel
-  className="pf-v6-c-title pf-m-lg ws-title-level-heading"
+  className="pf-v6-c-title pf-m-lg"
   asGrid
   styleProps={styleProps.third} />
-<div class="pf-v6-u-mt-lg">
-</div>
+<div class="pf-v6-u-mt-lg"></div>
 
 ### Fourth level heading (md, H4)
 
 <TitleLevel
-  className="pf-v6-c-title pf-m-lg ws-title-level-heading"
+  className="pf-v6-c-title pf-m-lg"
   asGrid
   styleProps = {styleProps.fourth} />
-<div class="pf-v6-u-mt-lg">
-</div>
+<div class="pf-v6-u-mt-lg"></div>
 
 ### Default body text
 
 **Note:** This is the default text style for paragraphs, lists, tables, etc. Some components instead use Red Hat Text at 700 font weight, which is Red Hat Text Medium (such as alerts and navigation).
 
 <TitleLevel
-  className="pf-v6-c-title pf-m-lg ws-title-level-heading"
+  className="pf-v6-c-title pf-m-lg"
   asGrid
   styleProps = {styleProps.body} />
-<div class="pf-v6-u-mt-lg">
-</div>
+<div class="pf-v6-u-mt-lg"></div>
 
 ### Small text
 
 <TitleLevel
-  className="pf-v6-c-title pf-m-lg ws-title-level-heading"
+  className="pf-v6-c-title pf-m-lg"
   asGrid
   styleProps = {styleProps.small} />
-<div class="pf-v6-u-mt-lg">
-</div>
-
-### Tiny text
-
-<TitleLevel
-  className="pf-v6-c-title pf-m-lg ws-title-level-heading"
-  asGrid
-  styleProps = {styleProps.tiny} />
-<div class="pf-v6-u-mt-lg">
-</div>
+<div class="pf-v6-u-mt-lg"></div>
 
 ### Code
 
 **Note:** Used for code blocks.
 
 <TitleLevel
-  className="pf-v6-c-title pf-m-lg ws-title-level-heading"
+  className="pf-v6-c-title pf-m-lg"
   asGrid
   styleProps = {styleProps.code} />
-<div class="pf-v6-u-mt-lg">
-</div>
+<div class="pf-v6-u-mt-lg"></div>
 
 ## Customizing heading levels
 If you use the [title component](/components/title) to create headings on your page, you can modify the default relationship between heading levels and text sizes. When using title, you must specify a heading level that will apply to the target text. The following table shows the default mapping of heading level to text size:
 
-| Heading level | Default size|
-|----|-----------|
-| H1 | 2xl (24px)|
-| H2 | xl (20px) |
-| H3 | lg (18px) |
-| H4 | md (16px) |
-| H5 | md (16px) |
-| H6 | md (16px) |
+| Heading level | Default size |
```

Comment:
Is there any way here to use the tokens rather than hard coding the px values? If it changes, this will be out of date.